packages/wp-story-editor/src/components/fontCheck/fontCheckDialog.js

```diff
@@ -0,0 +1,177 @@
+
+import {
+  useStory,
+  useConfig,
+  Dialog,
+  FontPicker,
+  TEXT_ELEMENT_DEFAULT_FONT,
+} from '@googleforcreators/story-editor';
+
+const DialogContent = styled.div`
+  display: grid;
+  row-gap: 21px;
+`;
+
+const ChevronRight = styled(Icons.ChevronRightSmall)`
+  width: 32px;
+  height: 32px;
+`;
+
+const ChevronLeft = styled(ChevronRight)`
+  transform: rotate(180deg);
+`;
+
+const DashboardLink = styled.span`
+  display: flex;
+  color: black;
+  align-self: flex-start;
+  align-items: center;
+  font-weight: normal;
+  margin-right: 40px;
+`;
+
+export const FontCheckDialog = ({
+  isOpen,
+  defaultCloseAction,
+  missingFont,
+  closeDialog,
+}) => {
+  const { dashboardLink, dashboardSettingsLink, isRTL } = useConfig();
+  const [suggestedFont, setSuggestedFont] = useState(TEXT_ELEMENT_DEFAULT_FONT);
+  const { updateElementsByFontFamily } = useStory(({ actions }) => ({
+    updateElementsByFontFamily: actions.updateElementsByFontFamily,
+  }));
+
+  const onSettingsClick = useCallback((evt) => {
+    trackClick(evt, 'click_font_settings');
+  }, []);
+
+  const updateMissingFontWithDefault = useCallback(() => {
+    updateElementsByFontFamily({
+      family: missingFont,
+      properties: { font: TEXT_ELEMENT_DEFAULT_FONT },
+    });
+    closeDialog();
+  }, [closeDialog, missingFont, updateElementsByFontFamily]);
+
+  const updateMissingFontWithSelected = useCallback(() => {
+    updateElementsByFontFamily({
+      family: missingFont,
+      properties: { font: suggestedFont },
+    });
+    closeDialog();
+  }, [suggestedFont, closeDialog, missingFont, updateElementsByFontFamily]);
+
+  const Chevron = isRTL ? ChevronRight : ChevronLeft;
+
+  return (
+    <Dialog
+      isOpen={isOpen}
+      onClose={defaultCloseAction}
+      title={__('Missing Fonts', 'web-stories')}
+      contentLabel={__('Missing Fonts', 'web-stories')}
+      actions={
+        <>
+          <Link href={dashboardLink}>
+            <DashboardLink>
+              <Chevron />
+              {__('Back to dashboard', 'web-stories')}
+            </DashboardLink>
+          </Link>
+          <Button
+            type={BUTTON_TYPES.SECONDARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={updateMissingFontWithDefault}
+          >
+            {__('Open anyway', 'web-stories')}
+          </Button>
+          <Button
+            type={BUTTON_TYPES.PRIMARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={updateMissingFontWithSelected}
+          >
+            {__('Replace font', 'web-stories')}
+          </Button>
+        </>
+      }
+    >
+      <DialogContent>
+        <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
+          {__(
+            'This story contains the following font that is no longer available:',
+            'web-stories'
+          )}
+        </Text>
+        <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL} isBold>
+          {missingFont}
+        </Text>
```

Comment:
So, @swissspidy correct me here but I think you can do this: 
```
<Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
{sprintf(
/* translators: %s: name of missing font */
__('This story contains the following font that is no longer available: %s',  'web-stories'),
`<strong>${missingFont}</strong>`
)
</Text>
```

Reply:
In theory yes. Jonny also mentioned this in a comment here somewhere. But since these two strings are both on their own line (i.e. `display: block`) and the `$missingFont` is more like a list item, it should be OK to keep these strings separate.

Appreciate your eye on that though!